plugin/metrics/disabled/metricsstore/reader_test.go

```diff
@@ -0,0 +1,74 @@
+// Copyright (c) 2021 The Jaeger Authors.
```

Comment:
> plugin/metrics/disabled/metricsstore/

a bit of overkill with nesting of packages, I would've kept it next to Factory under metrics/disabled